src/components/LectureEventModal/LectureEventModal.tsx

```diff
@@ -83,6 +85,18 @@ const LectureEventModal = ({
             </div>
           ))}
         </div>
+        {lectureEvent.title.toLowerCase().includes('anglų kalba') && (
+          <div className={styles.lectureInfoContainer}>
+            {' '}
+            <ReactSVG
+              src={popMasterrIcon}
+              className={styles.popMasterrLogoContainer}
+              onClick={() => {
+                window.open('https://popmasterr.netlify.app/  ', '_blank');
```

Comment:
"https://popmasterr.netlify.app", I removed the "/" and the white space at the end